docs/model_developers/modules.rst

```diff
@@ -329,3 +329,41 @@ for supplying the agents with the appropriate ``inmanta_plugins`` packages.
 The only exception to this rule is when using the ``inmanta export`` command. It exports a project and all its modules'
 ``inmanta_plugins`` packages to the orchestrator server. When this method is used, the orchestrator does not install any modules
 from the Python package repository but instead contains all Python code as present in the local Python environment.
+
+Configure a project to install modules from a private python package repository
+--------------------------------------------------------------------------------
+
+v2 modules are Python packages that can be installed from private Python package repositories, which may require authentication. This section explains the steps to install such modules.
+
+Create a file named ``/var/lib/inmanta/.netrc`` in the orchestrator's file system. 
+Add the following content to the file:
+
+.. code-block:: text
+
+  machine <url of the private repository>
+  login token
```

Comment:
Why not ?
```suggestion
  login <login>
```

Reply:
because that's how guillaume described it here: https://app.zenhub.com/workspaces/core-5a7c152b04a1652024289b7b/issues/gh/inmanta/inmanta-service-orchestrator/392 :p 
I haven't tested it yet

Reply:
it has been tested and token is indeed correct

Reply:
`token` is the username that is used by Cloudsmith when you use an entitlement token. I don't think we should document the procedure for Cloudsmith here. Customers probably want to connect to their own internal Python package repository that requires authentication, so I would keep it generic.
```suggestion
  login <username>
```